rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -715,6 +715,14 @@ protected NodeRunner buildNodeRunner() {
                 internalServices.add(getMinerClient());
             }
         }
+        if (getRskSystemProperties().isFlushEnabled()) {
```

Comment:
Have you evaluated to remove this flag? it seems to me that the full node is almost useless if it doesn't ever flush